scripts/linters/other_files_linter.py

```diff
@@ -289,8 +289,12 @@ def _check_that_workflow_steps_use_merge_action(
         """
         jobs_without_merge = []
         for job, job_dict in workflow_dict['jobs'].items():
+            # Ensure that atleast 1 job in a workflow uses the merge action.
```

Comment:
This isn't the intended behavior of this linter. We generally want to make sure that every job uses the merge action. I think your case is an exception because the job doesn't depend on the code, so we should have some kind of ignore list or ignore comment instead